pallets/parachain-staking/src/weights.rs

```diff
@@ -52,24 +51,235 @@ use sp_std::marker::PhantomData;
 
 /// Weight functions needed for parachain_staking.
 pub trait WeightInfo {
+	fn set_staking_expectations() -> Weight;
 	fn set_inflation() -> Weight;
+	fn set_parachain_bond_account() -> Weight;
+	fn set_parachain_bond_reserve_percent() -> Weight;
+	fn set_total_selected() -> Weight;
+	fn set_collator_commission() -> Weight;
+	fn set_blocks_per_round() -> Weight;
+	fn join_candidates(x: u32) -> Weight;
+	fn leave_candidates(x: u32) -> Weight;
+	fn go_offline() -> Weight;
+	fn go_online() -> Weight;
+	fn candidate_bond_more() -> Weight;
+	fn candidate_bond_less() -> Weight;
+	fn nominate(x: u32, y: u32) -> Weight;
+	fn leave_nominators(x: u32) -> Weight;
+	fn revoke_nomination() -> Weight;
+	fn nominator_bond_more() -> Weight;
+	fn nominator_bond_less() -> Weight;
 }
 
 /// Weights for parachain_staking using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+	fn set_staking_expectations() -> Weight {
+		(40_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(5 as Weight))
+			.saturating_add(T::DbWeight::get().writes(3 as Weight))
+	}
 	fn set_inflation() -> Weight {
-		(20_169_000 as Weight)
+		(84_000_000 as Weight)
```

Comment:
weights generated on my 8-core mac are ~4x larger than weights generated on the production hardware

Reply:
it varies, see https://github.com/PureStake/moonbeam/pull/525/commits/8672b011f9ece195aecf6e1a3a5af61a7ca80a3e